br/pkg/task/backup_ebs.go

```diff
@@ -253,6 +273,123 @@ func RunBackupEBS(c context.Context, g glue.Glue, cmdName string, cfg *BackupEBS
 	return nil
 }
 
+func waitAllScheduleStoppedAndNoRegionHole(ctx context.Context, cfg Config, mgr *conn.Mgr) error {
+	allStores, err := conn.GetAllTiKVStoresWithRetry(ctx, mgr.GetPDClient(), util.SkipTiFlash)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	// we wait for 15*40 = 600s = 10m at least
+	for i := 0; i < 40; i++ {
+		if ctx.Err() != nil {
+			return ctx.Err()
+		}
+		if i > 0 {
+			time.Sleep(waitAllScheduleStoppedInterval)
+		}
+		allRegions, err2 := waitUntilAllScheduleStopped(ctx, cfg, allStores, mgr)
+		if err2 != nil {
+			if causeErr := errors.Cause(err2); causeErr == hasPendingAdminErr {
+				log.Info("schedule ongoing on tikv, will retry later", zap.Error(err2))
+			} else {
+				log.Warn("failed to wait schedule, will retry later", zap.Error(err2))
+			}
+			continue
+		}
+
+		// sort by start key
+		sort.Slice(allRegions, func(i, j int) bool {
+			left, right := allRegions[i], allRegions[j]
+			return bytes.Compare(left.StartKey, right.StartKey) < 0
+		})
+
+		var hasHole bool
+		for j := 0; j < len(allRegions)-1; j++ {
+			left, right := allRegions[i], allRegions[j]
+			// todo: do we need to prefix every key with 'z'?, seems doesn't affect sorting and checking
+			if bytes.Compare(left.EndKey, right.StartKey) != 0 {
+				log.Info("region hole found", zap.Reflect("left-region", left), zap.Reflect("right-region", right))
```

Comment:
backup stuck here:
`I0912 01:05:12.897585       9 backup.go:267] [2022/09/12 01:05:12.897 +00:00] [INFO] [backup_ebs.go:345] ["region hole found"] [left-region="{\"id\":612,\"start_key\":\"dIAAAAAAAAD/TV9pgAAAAAD/AAABA4AAAAD/AAAABQOAAAD/AAAAAAMDgAD/AAAAAAeIA4D/AAAAAAAABgD+\",\"end_key\":\"dIAAAAAAAAD/TV9pgAAAAAD/AAABA4AAAAD/AAAABQOAAAD/AAAAAAcDgAD/AAAAAAqsA4D/AAAAAAAABAD+\",\"region_epoch\":{\"conf_ver\":5,\"version\":65},\"peers\":[{\"id\":613,\"store_id\":1},{\"id\":614,\"store_id\":4},{\"id\":615,\"store_id\":5}]}"] [right-region="{\"id\":8,\"end_key\":\"dIAAAAAAAAD/BQAAAAAAAAD4\",\"region_epoch\":{\"conf_ver\":5,\"version\":2},\"peers\":[{\"id\":9,\"store_id\":1},{\"id\":10,\"store_id\":4},{\"id\":11,\"store_id\":5}]}"]`

Reply:
region 612
`"region_id": 612,
      "region_local_state": {
        "region": {
          "id": 612,
          "start_key": "7480000000000000FF4D5F698000000000FF0000010380000000FF0000000503800000FF0000000003038000FF0000000007880380FF0000000000000600FE",
          "end_key": "7480000000000000FF4D5F698000000000FF0000010380000000FF0000000503800000FF0000000007038000FF000000000AAC0380FF0000000000000400FE",`

region  id 8
```
"region_id": 8,
      "region_local_state": {
        "region": {
          "id": 8,
          "start_key": "",
          "end_key": "7480000000000000FF0500000000000000F8",
```

Reply:
the log is misleading since the bug below.